<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DecoderObject</class>
 <widget class="QDialog" name="DecoderObject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Decoder</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QPushButton" name="EncodeButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 127);font-size:15px;</string>
   </property>
   <property name="text">
    <string>Encode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DecodeButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 127);font-size:15px;</string>
   </property>
   <property name="text">
    <string>Decode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ResetButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 127);font-size:15px;</string>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 127);font-size:15px;</string>
   </property>
   <property name="text">
    <string>Plain Text</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 127);font-size:15px;</string>
   </property>
   <property name="text">
    <string>Coded Text</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FirstLineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 0, 127);font-size:15px</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SecondLineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 127);font-size:15px;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
